---
layout: default
title: Curriculum Vitae
---
<!-----
NEW: Check the "Suppress top comment" option to remove this info from the output.

Conversion time: 0.599 seconds.


Using this Markdown file:

1. Paste this output into your source file.
2. See the notes and action items below regarding this conversion run.
3. Check the rendered output (headings, lists, code blocks, tables) for proper
   formatting and use a linkchecker before you publish this page.

Conversion notes:

* Docs to Markdown version 1.0β30
* Tue Aug 24 2021 11:10:48 GMT-0700 (PDT)
* Source doc: working CV
* This is a partial selection. Check to make sure intra-doc links work.

WARNING:
You have 9 H1 headings. You may want to use the "H1 -> H2" option to demote all headings by one level.

----->


<p style="color: red; font-weight: bold">>>>>>  gd2md-html alert:  ERRORs: 0; WARNINGs: 1; ALERTS: 0.</p>
<ul style="color: red; font-weight: bold"><li>See top comment block for details on ERRORs and WARNINGs. <li>In the converted Markdown or HTML, search for inline alerts that start with >>>>>  gd2md-html alert:  for specific instances that need correction.</ul>

<p style="color: red; font-weight: bold">Links to alert messages:</p>
<p style="color: red; font-weight: bold">>>>>> PLEASE check and correct alert issues and delete this message and the inline alerts.<hr></p>



# Education

**<span style="text-decoration:underline;">University of British Columbia, Fall 2017 - Present</span>**

Pursuing a PhD in Bioinformatics, Advised by Dr. Sara Mostafavi and Dr. Jessica Dennis

**<span style="text-decoration:underline;">Carnegie Mellon University, Class of 2017</span>**

BS in Computational Biology, School of Computer Science & Mellon College of Science


# University Employment

**<span style="text-decoration:underline;">Fall 2018 - Winter 2019 Course Assistant BIOF 540, University of British Columbia</span>**

**	**Role: Revising course materials and running tutorial sections for the statiscal 


    methods for high dimensional biology course at the University of British Columbia

	Supervised by: Sara Mostafavi, Bioinformatics, CMMT (saram@stat.ubc.ca)


# Research Experience

**<span style="text-decoration:underline;">Spring 2020 - Present, Graduate Student Mostafavi and Dennis Lab</span>**

**	**“Sex-Specific Regulatory Effects in Placenta”

	Interests: Methylation QTL analysis, TWAS

	Advised by:

		Sara Mostafavi, UBC Statistics, Bioinformatics, CMMT (saram@stat.ubc.ca)


    Jessica Dennis, UBC Medical Genetics, BCCHRI (jessica.dennis@bcchr.ca)

**<span style="text-decoration:underline;">Fall 2017 - Spring 2020, Graduate Student Mostafavi Lab</span>**

**	**“Causal Mediation of Multiple Genotype Effects in Multi-Omic Data”

	Interests: Integration of Epigenetic, Genetic, and Transcriptional Information

Advised by:


    Sara Mostafavi, UBC Statistics, Bioinformatics, CMMT (saram@stat.ubc.ca)

**<span style="text-decoration:underline;">May - August 2018, NSERC-CREATE Fellowship, Kaufer lab, UC Berkeley</span>**

**	**“Joint Modelling of Rat Behavior and Oligodendrocyte Formation in Hippocampus, with Insight into Genomic ”

	Advised by:

		Daniela Kaufer, Helen Wills Neuroscience Institute (danielak@berkeley.edu)

**<span style="text-decoration:underline;">Summer 2016 - Spring 2017, Research Assistant, Carnegie Mellon University</span>**

“Maximum Parsimony Phylogenies using Generalized Buneman Pruning,” 

Advised by:


    Russell Schwartz, Dept. of Computational Biology (russells@andrew.cmu.edu)

**<span style="text-decoration:underline;">Summer 2015 - Fall 2015,  Researcher CMU iGEM</span>**

“Carnegie Mellon iGEM Biosensor Emission Analysis Machine (BEAM),”

Advised by:


    Cheryl Telmer, MBIC, Carnegie Mellon University (ctelmer@andrew.cmu.edu)

**<span style="text-decoration:underline;">Fall 2014, Research Assistant, Carnegie Mellon University</span>**

“Unmixing Bulk Tumor Data with Simplical Complexes”

Advised by:


    Russell Schwartz, Dept. Computational Biology (russells@andrew.cmu.edu)

**<span style="text-decoration:underline;">Fall 2013 - Spring 2014, Student Researcher, Carnegie Mellon University</span>**

“HHMI SEAPHAGES Program”,

Advised by:


    Maggie Braun, Dept. Biology (mabraun@andrew.cmu.edu)


    Javier Lopez, Dept. Biology (jlaa@andrew.cmu.edu)


    Joel McManus, Dept. Biology (mcmanus@andrew.cmu.edu)


# Publications and Contributions


    Wright, Galen E. B., Nicholas S. Caron, Bernard Ng, Lorenzo Casal, **William Casazza**, Xiaohong Xu, Jolene Ooi, et al. “Gene Expression Profiles Complement the Analysis of Genomic Modifiers of the Clinical Onset of Huntington Disease.” Human Molecular Genetics. https://doi.org/10.1093/hmg/ddaa184.


    Patrick E, Taga M, Ergun A, Ng B, **Casazza W**, et al. (2020) Deconvolving the contributions of cell-type heterogeneity on cortical gene expression. PLOS Computational Biology 16(8): e1008120. https://doi.org/10.1371/journal.pcbi.1008120


    Long, Kimberly L. P., Jocelyn P. Breton, Linda Chao, Siamak Sorooshyari, Kelsey Hu, Anjile An, Yurika Kazama, **et al**. “Region-Specific Maladaptive Myelination Contributes to Differential Susceptibility to Stress-Induced Avoidance and Acute Threat Reactivity in Humans and Rodents.” Biological Psychiatry 87, no. 9, Supplement (2020): S88. https://doi.org/10.1016/j.biopsych.2020.02.247.


    Ng, Bernard, **William Casazza**, Ellis Patrick, Shinya Tasaki, Gherman Novakovsky, Daniel Felsky, Yiyi Ma, et al. “Using Transcriptomic Hidden Variables to Infer Context-Specific Genotype Effects in the Brain.” The American Journal of Human Genetics 105, no. 3 (September 5, 2019): 562–72. https://doi.org/10.1016/j.ajhg.2019.07.016.


    [Klyczek, Karen K., et al. “Tales of Diversity: Genomic and Morphological Characteristics of Forty-Six Arthrobacter Phages.” PloS One, vol. 12, no. 7, July 2017, p. E0180517.](http://paperpile.com/b/VN1328/fven)


    Hwang, Yih-Chii. "A Cloud based Framework for Benchmarking Variant Calling Tools for High throughput Sequencing Data in the Human Genome." Poster presented at: ICBI symposium. 27 October 2017. Georgetown University


    Open review article: Asthana, Ruchi et al. “IGEM RESEARCH ARTICLE: Development and Characterization of Fluorescent and Luminescent Biosensors for Estrogenic Activity.” PLOS Collections, 24 Aug. 2016, blogs.plos.org/collections/igem-research-article-001/.


# Awards and Recognitions

NSERC-CREATE Scholarship Award, 2017-2018

UBC Faculty of Science Graduate Award, 2017-2021

2nd Place HackSeq 2017 UBC, 2017

HHMI Summer Undergraduate Research Grant, 2014


# Industry Experience

**<span style="text-decoration:underline;">Scientist Intern at DNAnexus Inc., May 2017-August 2017</span>**

**	**“Automatic Cloud-Based Benchmarking of Structural Variant Calling Software.”

Supervised by: 


    Andrew Carroll (acarroll@dnanexus.com)


        Yih-Chii Hwang (yihhwang@dnanexus.com)


# Activities

**Volunteering**

Let’s talk Science: Python Coding Workshop, Summer 2019

BC Children’s Research Open House, Fall and Spring 2018, Spring 2019

Carnegie Mellon Global Medical Brigades, 2013-2014

**Extracurriculars**

Vancouver Bioinformatics User Group (VanBUG) Organizing Committee, Fall 2018-Present

Trainee Omics Group (TOG) Study Group Lead, Spring 2019-Present


# Research Interests

Integration of Multi-omic Data 

Complex Traits

Gene by Environment Interaction

Sex-Specific Quantitative Trait Loci 

Causal Inference in Observational Data


# References


    Dr. Sara Mostafavi , Room 3021, Centre for Molecular Medicine and Therapeutics, 950 West 28th Avenue, Vancouver, BC V5Z 4H4 Canada, 604-875-3802, saram@stat.ubc.ca


    Dr. Jessica Dennis, Room 371A, BC Childrens Hospital Research Institute, 920 West 28th Avenue, Vancouver, BC V5Z 4H4 Canada,  604-875-7871, jessica.dennis@bcchr.ca


    Dr. Russell Schwartz, 4400 Fifth Avenue, Mellon Institute 654B, Pittsburgh, PA 15213 USA, 412-268-3971, russells@andrew.cmu.edu

